commons/protocol/src/main/scala/sbt/protocol/Protocol.scala

```diff
@@ -682,6 +686,8 @@ object Message {
   private implicit val sendSyntheticBuildChangedUnpickler = genUnpickler[SendSyntheticBuildChanged]
   private implicit val sendSyntheticValueChangedPickler = genPickler[SendSyntheticValueChanged]
   private implicit val sendSyntheticValueChangedUnpickler = genUnpickler[SendSyntheticValueChanged]
+  private implicit val daemonRequestPickler = genPickler[DaemonRequest]
```

Comment:
it should use the ReceivedResponse ("ack"), though I don't remember whether I send that...